basxconnect/mailer_integration/views.py

```diff
@@ -1,5 +1,6 @@
 import traceback
 
+import bread.layout.components.forms.helpers
```

Comment:
Not sure why we need an import here. Was this missing before?

Reply:
No, that was just an experiment to find out why the IDE was complaining. But since it works without the import this seems to be a bug in PyCharm
![image](https://user-images.githubusercontent.com/11404846/148188326-fc1ebb49-f4a3-4e26-a34b-54b152187eb9.png)